content/conditions/laryngitis/do.md

```diff
@@ -0,0 +1,17 @@
+### Do
+
+- try to speak as little as possible
+- drink plenty of fluids
+- keep the air moist by putting out bowls of water or using a humidifier -
+  central heating and air conditioning make the air dry
+- gargle with warm salty water
+{.list--check}
+
+::: reveal How to gargle with salty water
+1. Dissolve half a teaspoon of salt in a glass of warm water - warm water helps
+   salt dissolve.
+1. Gargle with the solution then spit it out - don’t swallow it.
```

Comment:
Should we use the right sequence since MD is meant to be readable as plain text?